ninja/signature/details.py

```diff
@@ -241,6 +242,8 @@ def _get_param_type(self, name: str, arg: inspect.Parameter) -> FuncParam:
 
 def is_pydantic_model(cls: Any) -> bool:
     try:
+        if get_origin(cls) == Union:
+            return all([issubclass(arg, pydantic.BaseModel) for arg in get_args(cls)])
```

Comment:
Hello 😄 

You can delete useless `[]` for list comprehension on using all()

```suggestion
            return all(issubclass(arg, pydantic.BaseModel) for arg in get_args(cls))
```


```python
In [1]: l = [1, 2, 3]

In [2]: all(ele for ele in l)
Out[2]: True

In [3]: all([ele for ele in l])
Out[3]: True
```